apps/docs/components/reference/RefFunctionSection.tsx

```diff
@@ -130,6 +130,14 @@ const RefFunctionSection: React.FC<IRefFunctionSection> = (props) => {
                   {item.examples &&
                     item.examples.map((example, exampleIndex) => {
                       const exampleString = ''
+
+                      const codeBlockLang = example?.code?.includes('```js')
+                        ? 'js'
+                        : example?.code?.includes('```ts')
+                        ? 'ts'
+                        : example?.code?.includes('```dart')
+                        ? 'dart'
+                        : 'js'
```

Comment:
Are we able to use `.startsWith()` here instead of `.includes()`? Shouldn't be an issue really but might make it slightly safer 😅

Reply:
@alaister just changed this now to use `.startsWith()`